jib-core/src/main/java/com/google/cloud/tools/jib/http/BlobHttpContent.java

```diff
@@ -59,7 +59,7 @@ public String getType() {
 
   @Override
   public boolean retrySupported() {
-    return false;
+    return blob.isRetryable();
```

Comment:
Continuing the discussion, you'll see that now this package `c.g.c.t.jib.http` is the abstraction layer where we can start talking about things around HTTP.